### PR TITLE
Update Sveltekit starter package dependencies. Vite 4 Svelte 4

### PR DESCRIPTION
**Pull request by zavx0z**
The dependencies in the `package-lock.json` for the Sveltekit starter project have been updated. This includes an update to `"lockfileVersion": 3` and bumping various dependencies to their latest versions such as `"@dfinity/agent": "^0.18.1"`, `"@sveltejs/kit": "^1.20.4` etc. Additionally, a new dependency `"@playwright/test": "^1.28.1"` has been added. It appears that various development tools and libraries have been updated to their latest version for better performance, compatibility, and security.

### CODE REVIEW

File: svelte/sveltekit-starter/dfx.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
 			"packtool": ""
 		}
 	},
-	"dfx": "0.12.0",
+	"dfx": "0.14.3",
```
Comment (sesi200) [inline svelte/sveltekit-starter/dfx.json:22]:
```suggestion
```
I'd say removing that line is even better so we don't have to update the version all the time

**Review by sesi200 — COMMENTED** [review]
Does the project deploy properly for you with dfx 0.14.3? I only see a bunch of errors:

**Comment by sesi200** [comment]
I still get the same errors

**Comment by dfx-json** [comment]
thank you for your contribution. I will be closing this PR as it does not seem that we will be able to move forward. if you would still like to submit your changes, please do so in a new PR. thank you for your understanding.

**Pull request closed without merging**